docker-compose.yml

```diff
@@ -624,6 +624,37 @@ services:
       - triplestore:virtuoso
     logging: *default-logging
 
+################################################################################
+  # CLB CONSUMERS
+  ################################################################################
+  clb-contact-data-consumer:
+    image: lblod/delta-consumer:0.0.17-rc.7
```

Comment:
**Question:** I noticed we are currently using a mix of 0.0.17-rc.7 and 0.0.18 of the `delta-consumer` service. Is there a reason you opted for this version?

Reply:
Not really I just copied another consumer config, I updated it to 0.0.18 as it seems to be a newer version without any breaking changes